charts/firefly/Chart.yaml

```diff
@@ -19,7 +19,7 @@ name: firefly
 description: A Helm chart for deploying FireFly and FireFly HTTPS Dataexchange onto Kubernetes.
 type: application
 appVersion: "1.0.0"
-version: "0.5.0"
+version: "0.5.1"
```

Comment:
No need to bump it yet, we haven't released `0.5.0` it got bumped as part of #38 (nor do we have version checker enabled with `ct`).

We're just waiting for FireFly `v1.0.0` to drop.